pkg/webhook/policy_test.go

```diff
@@ -606,6 +618,15 @@ func TestGetDenyMessages(t *testing.T) {
 			},
 			ExpectedMsgCount: 1,
 		},
+		{
+			Name: "One Dry Run, One Deny, One Warn",
+			Result: []*rtypes.Result{
+				resDryRun,
+				resDeny,
+				resWarn,
+			},
+			ExpectedMsgCount: 2,
```

Comment:
We should also test that the correct message is going to the correct response field, event type and log line.